### PR TITLE
Population cleaning

### PR DESCRIPTION
**Pull request by DivyaSharma0795**
Added Population Data from cdc -
1. Downloaded source files from the website and placed them in the "00_Source_Data" folder
2. Cleaned the data in the 'Population_Cleaning.ipynb' file

### CODE REVIEW

**Comment by JiayiZhou36** [comment]
Can the code convert to .py format? It is a little difficult to review in jupyter notebook.

**Comment by DivyaSharma0795** [comment]
> Can the code convert to .py format? It is a little difficult to review in jupyter notebook.

I have added a .py file for your review

**Comment by JiayiZhou36** [comment]
Look good to merge!

File: 10_Code/Population_Cleaning.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+"""
+The population data has been taken from https://wonder.cdc.gov/bridged-race-population.html
+The data is in the form of tab-separated values, with additional info towards the end of the document, starting with the text "---"
+We have downloaded the data from 1990 to 2010 separately, and then 2010 to 2020 separately, to maintain the file sizes and ensure complete download of data. We will combine the two in this code.
+We will also remove the additional info at the end of the document while reading the file.
+The raw downloaded data has been saved in the folder "00_raw_data" as "pop_1990_2010.txt" and "pop_2010_2020.txt" 
+"""
+# Importing libraries
```
Comment (carrieli15) [inline 10_Code/Population_Cleaning.py:8]: I like your coding habit of putting all the libraries first. I may import the library in the middle of the code chuck just for my convenience. But when I act in the role of reviewer I realize that put all the libraries at the beginning is good for others to understand the whole picture of the code.

File: 10_Code/Population_Cleaning.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+reader = pd.read_csv(data_2010_2020, iterator=True, chunksize=1, sep="\t")
+for chunk in reader:
+    if stop_reading(chunk.to_string()):
+        df_2010_2020 = pd.concat([df_2010_2020, chunk])
+    else:
+        break
+
+df_1990_2010.shape
+# (66129, 8)
```
Comment (carrieli15) [inline 10_Code/Population_Cleaning.py:41]: Put some important results as comments is also my take-away from your code. It is also helpful for me to understand your code.

**Review by carrieli15 — COMMENTED** [review]
The script is well-structured and follows a logical sequence of operations to achieve the stated goal.

**Pull request merged**